clocksilencer.ino

```diff
@@ -169,3 +178,15 @@ void startPlayer() {
   myDFPlayer.volume(25);
   delay(10);
 }
+
+DateTime grabTime(){
+  digitalWrite(14,HIGH);
+  if (!rtc.begin()) {
+    Serial.println("Couldn't find RTC");
+    while (1);
```

Comment:
This is something new to me. Isn't

```c++
while(1);
```
an infinite loop?

I would rather expect sth like:


```c++
while (!rtc.begin()) {
    Serial.println("Waiting for RTC...");
    delay(100);
}
```

Reply:
It is. But when RTC cannot be found, we should not go further. This is critical error.

Reply:
ok, but don't we want to retry? I think current version is checking it only once and then going into infinite `while` loop. and we will not be notified and we will use the battery...

maybe we can react better to the critical error e.g.:

1. ding-dong forever until someone comes and resets the hardware
2. go into deep sleep for 20 minutes
3. ???

but if you don;t want to change that maybe just the comment explaining why we are doing so?

Reply:
Hmm, Deepsleep would be nice solution. Good idea.

Reply:
I presume that silence would be alarming enough. Deepsleep is a good solution.